src/Models/Tile.php

```diff
@@ -307,51 +327,54 @@ public function canEdit($member = null) {
             return $extended;
         }
 
-        // check for inherit
+        // fail if type === Inherit & member cannot edit parent
         if ($this->CanEditType == 'Inherit') {
-            if (!$this->ParentID) {
-                return true;
+            // first check if it has a parent (it always should?)
+            if ($this->ParentID) {
+                $parentCanEdit = DataObject::get_by_id(TileElement::class, $this->ParentID)->canEdit($member);
+                if ($parentCanEdit === false) {
+                    return false;
+                }
             }
-            return DataObject::get_by_id(TileElement::class, $this->ParentID)->canEdit($member);
         }
 
-        // check for any logged-in users
-        if ($this->CanEditType == 'LoggedInUsers' && $member) {
-            return true;
+        // fail if type === LoggedInUsers & memberID === false
+        if ($this->CanEditType == 'LoggedInUsers' && !$memberID) {
```

Comment:
If they are logged in but can't edit the parent page Then we are showing the Tiles edit button but we cant actually go to this edit page?